GoogleDataLogger/GoogleDataLogger/DependencyWrappers/GDLConsoleLogger.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "GULLogger.h"
+
+/** The console logger prefix. */
+static GULLoggerService kGDLConsoleLogger = @"[GoogleDataLogger]";
+
+/** A list of message codes to print in the logger that help to correspond printed messages with
+ * code locations. */
+typedef NS_ENUM(NSInteger, GDLMessageCode) {
+
+  /** For warning messages concerning transform: not being implemented by a log transformer. */
+  GDLMCWTransformerDoesntImplementTransform = 1
```

Comment:
What does MCW stand for?

Reply:
MessageCodeWarning. I'm trying to keep these strings short. Any alternative suggestions?

Reply:
Probably fine with a comment.